packages/swirl-components/src/components/swirl-popover-trigger/swirl-popover-trigger.tsx

```diff
@@ -98,13 +110,46 @@ export class SwirlPopoverTrigger {
   };
 
   private onMouseleave = () => {
+    clearTimeout(this.hoverDelayReference);
+    // eslint-disable-next-line @stencil/strict-boolean-conditions
```

Comment:
Nitpick: You can prevent this by doing `if (!Boolean(this.hoverDelayReference))`